### PR TITLE
revise ipynb tutorials

### PR DESCRIPTION
**Pull request by lidongze0629**
## What do these changes do?

- [x] Code format for tutorial 'ipynb' 

## Related issue number

Fixes

### CODE REVIEW

File: tutorials/zh/7_unsupervised_learning_with_graphsage.ipynb [hunk 1]
```diff
@@ -29,7 +29,8 @@
    "source": [
     "# Install graphscope package if you are NOT in the Playground\n",
     "\n",
-    "!pip3 install graphscope"
+    "!pip3 install graphscope\n",
+    "!pip3 uninstall -y importlib_metadata  # Address an module conflict issue on colab.google. Remove this line if you are not on colab."
```
Comment (sighingnow) [inline tutorials/zh/7_unsupervised_learning_with_graphsage.ipynb:33]: Why there's a conflict?

**Review by sighingnow — CHANGES_REQUESTED** [review]
Please split your PR into two PR, as the title is "resolve an module conflict issue", but you do a lot notebook changes and the diff that belongs to the title is totally lost.

You could have a "revisit the tutorial notebooks" PR separately.

File: tutorials/zh/7_unsupervised_learning_with_graphsage.ipynb (hunk 1 above)
Comment (lidongze0629) [reply tutorials/zh/7_unsupervised_learning_with_graphsage.ipynb:33]: Some error info like `'posixpath' object has no attribute 'read_text'` in `importlib_metadata.__init__` during `import tensorflow (from import graphlearn)` in our coordinator

**Comment by yecol** [comment]
> Please split your PR into two PR, as the title is "resolve an module conflict issue", but you do a lot notebook changes and the diff that belongs to the title is totally lost.
> 
> You could have a "revisit the tutorial notebooks" PR separately.

Yes, I agree.

**Review by yecol — CHANGES_REQUESTED** [review]
please revise following Tao's suggestion.

**Pull request merged**